lib/openhab/dsl/rules/property.rb

```diff
@@ -30,6 +32,7 @@ def prop(name)
               elsif block
                 instance_variable_set(:"@#{name}", block)
```

Comment:
@ccutrer, This is unrelated to the PR, but I just noticed this. Shouldn't we call the block here instead?

```suggestion
                instance_variable_set(:"@#{name}", block.call(name))
```

Reply:
I'm guessing that the idea was to pass a block to the property, e.g.
```
rule do
  tags { # build and return the required tags  }
end
```
Since this isn't documented I doubt anyone uses it this way.

Reply:
it would make sense to assign the block itself to the ivar, _if_ you then actually call that block when reading the property, and if there was actually some advantage to deferring execution of that block until the property is read, but that's not really the case. execution could be deferred, but only until the end of the `rule` block, which doesn't seem useful. and like you said, not document. so I vote we just remove this functionality completely